### ISSUE TITLE
table options without values are not parsed properly.

### ISSUE THREAD

**Issue by JustinAzoff** [zeek/zeekscript#41]
Looks like the ```&broker_allow_complex_type``` is what breaks things here.

```
$ cat foo.zeek 
module Foo;

global foo: table[string] of set[count] &broker_allow_complex_type &backend=Broker::MEMORY;
```

```
$ zeek -a foo.zeek
$
```

```
$ zeek-script --version
1.1.0
```

```
$ cat foo.zeek | zeek-format 
module Foo;

global foo: table[string] of set[count]&
```
```
$ cat foo.zeek | zeek-format | zeek-format
module Foo;
```

**Comment by bbannier** [comment]
Is this still an issue with 1.2.0? For your file I see the following

```console
$ zeek-format -v
1.2.0
```
```console
$ zeek-format foo.zeek
module Foo;

global foo: table[string] of set[count] &broker_allow_complex_type
    &backend=Broker::MEMORY;
```

**Comment by bbannier** [comment]
Closing this as I believe it was fixed by https://github.com/zeek/tree-sitter-zeek/commit/41494c8dc13930754a3ab8b72520bf6562fa29c7 which landed in this repo with 9f48f8912c7f157e082d3d0ddfd9d749d9bb7592.
